business-central-parent/business-central-webapp/pom.xml

```diff
@@ -2839,10 +2839,16 @@
         <artifactId>spotbugs-maven-plugin</artifactId>
       </plugin>
       <plugin>
-        <groupId>io.thorntail</groupId>
-        <artifactId>thorntail-maven-plugin</artifactId>
+        <groupId>org.wildfly.plugins</groupId>
+        <artifactId>wildfly-jar-maven-plugin</artifactId>
         <configuration>
-          <filterWebInfLib>uberjar-only</filterWebInfLib>
+<!--          <cli-sessions>-->
+<!--            <cli-session>-->
+<!--              <script-files>-->
+<!--                <script>../scripts/script.cli</script>-->
+<!--              </script-files>-->
+<!--            </cli-session>-->
+<!--          </cli-sessions>-->
```

Comment:
These will likely be removed and instead we pass in the script.cli during startup.